### PR TITLE
[CIVIS] Update DORA Metrics docs about querying

### PR DESCRIPTION
**Pull request by AdrianLC**
### What does this PR do? What is the motivation?
Explains how to query DORA Metrics and events outside of the provided page.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/dora_metrics/_index.md [hunk 1]
```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
...
+- `dora.incidents_impact` of type `count`. User for Change Failure Rate. It tracks the services or teams impacted by incidents. Used for Change Failure Rate with the formula `dora.incidents_impact / dora.deployments.count`. A big time rollup of at least 1 week is recommended to account for time difference between deployments and when the impact starts.
+- `dora.time_to_restore` of type `distribution`. Contains the time in `seconds` between the incident's `started_at` and `finished_at`.
+
+All the metrics contain the following tags when available:
+- `service`
+- `team`
+- `env`
+- `repository_id`
+
+Additionally `dora.incidents_impact` and `dora.time_to_restore` have the tag `severity`, if it was provided through the API.
```
Comment (andrea-mosk) [inline content/en/dora_metrics/_index.md:298]: We can probably exclude this, as it's available in the metrics explorer already maybe?

File: content/en/dora_metrics/_index.md [hunk 2]
```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
...
+Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][9].
+
+The metrics can also be queried with the API endpoints [Query timeseries points][10] and [Query timeseries data across multiple products][11].
+
+The metrics provided by DORA Metrics are:
+- `dora.deployments.count` of type `count`. Used for Deployment Frequency.
+- `dora.change_lead_time` of type `distribution`. Contains the age in `seconds` of the git commits at the time of deployment.
+- `dora.incidents_impact` of type `count`. User for Change Failure Rate. It tracks the services or teams impacted by incidents. Used for Change Failure Rate with the formula `dora.incidents_impact / dora.deployments.count`. A big time rollup of at least 1 week is recommended to account for time difference between deployments and when the impact starts.
```
Comment (andrea-mosk) [inline content/en/dora_metrics/_index.md:289]:
```suggestion
- `dora.incidents_impact` of type `count`. Used for Change Failure Rate. It tracks the services or teams impacted by incidents. Used for Change Failure Rate with the formula `dora.incidents_impact / dora.deployments.count`. A big time rollup of at least 1 week is recommended to account for time difference between deployments and when the impact starts.
```

Also we said twice "Used for Change Failure Rate". Maybe we can remove the first sentence?

File: content/en/dora_metrics/_index.md [hunk 3]
```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
...
 You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [DORA Metrics page][8].
 
-Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards and alerted on using [metric monitors][9].
+Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][9].
+
+The metrics can also be queried with the API endpoints [Query timeseries points][10] and [Query timeseries data across multiple products][11].
+
+The metrics provided by DORA Metrics are:
+- `dora.deployments.count` of type `count`. Used for Deployment Frequency.
+- `dora.change_lead_time` of type `distribution`. Contains the age in `seconds` of the git commits at the time of deployment.
```
Comment (andrea-mosk) [inline content/en/dora_metrics/_index.md:288]: Do we have a different page describing exactly how this is computed? If we do, I would reference that. If we don't, I would expect customers to ask about this in future

File: content/en/dora_metrics/_index.md (hunk 1 above)
Comment (AdrianLC) [reply content/en/dora_metrics/_index.md:298]: I would keep it. The metric names are also available in the explorer but we still want to list them here

**Review by drichards-87 — APPROVED** [review]
Thanks @AdrianLC. I've approved the PR. Can you confirm that this is ready to be merged (the merge check box is empty). Thanks!

**Comment by AdrianLC** [comment]
> Thanks @AdrianLC. I've approved the PR. Can you confirm that this is ready to be merged (the merge check box is empty). Thanks!

Thank you! Let's wait until @kassenq approves

File: content/en/dora_metrics/_index.md [hunk 4]
```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
 
 You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [DORA Metrics page][8].
 
-Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards and alerted on using [metric monitors][9].
+Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][9].
+
+The metrics can also be queried with the [Query timeseries points][10] and [Query timeseries data across multiple products][11] API endpoints.
+
+The metrics provided by DORA Metrics are:
```
Comment (kassenq) [inline content/en/dora_metrics/_index.md:286]: Yes, can we turn this list into a table with 3 columns: metric name, metric type, and description?

File: content/en/dora_metrics/_index.md [hunk 5]
```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
...
+
+All the metrics contain the following tags when available:
+- `service`
+- `team`
+- `env`
+- `repository_id`
+
+**Note**: The `severity` tag is available for the `dora.incidents_impact` and `dora.time_to_restore` metrics, if provided through the API.
+
+### Deployment and Incident events
```
Comment (kassenq) [inline content/en/dora_metrics/_index.md:300]: "Deployment and incident events"

File: content/en/dora_metrics/_index.md [hunk 6]
```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
...
+
+DORA Metrics also provides individual `deployment`, `incident`, and `incident_finished` events in [Event Management][12] with `source:software_delivery_insights`.
+
+The events can be queried and visualized with [Event Management][13].
```
Comment (kassenq) [inline content/en/dora_metrics/_index.md:304]: "The events can be queried and visualized with the [Events Explorer](https://docs.datadoghq.com/service_management/events/explorer/)."

**Review by kassenq — CHANGES_REQUESTED** [review]

File: content/en/dora_metrics/_index.md [hunk 7]
```diff
@@ -279,7 +279,32 @@ Events can be sent both at the start of and after incident resolution. Incident
...
 You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [DORA Metrics page][8].
 
-Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards and alerted on using [metric monitors][9].
+Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][9].
+
+The metrics can also be queried with the [Query timeseries points][10] and [Query timeseries data across multiple products][11] API endpoints.
+
+The metrics provided by DORA Metrics are:
+
+| Metric | Type | |
```
[the remark this answers is not in the document]
Comment (andrea-mosk) [reply content/en/dora_metrics/_index.md]: Added!

**Comment by andrea-mosk** [comment]
@drichards-87 Hey, this is good to be merged now!

**Pull request merged**